components/automate-ui/src/app/helpers/jwt/jwt.ts

```diff
@@ -19,6 +19,11 @@ export interface IDToken {
   email_verified: boolean;
   name: string;
   groups: Array<string>;
+  federated_claims: FederatedClaims;
+}
+
+interface FederatedClaims {
+  connector_id: string;
```

Comment:
Adding the `federated_claims.connector_id` to decern if the logged-in user is a local user or non-local user. Example values for this field are 'local' and 'saml'.